core/views/__init__.py

```diff
@@ -88,13 +88,16 @@ def can_edit(obj: Any, user: User):
     """Can the user edit the object.
 
     Args:
+
```

Comment:
La ligne vide n'est pas dans la norme de dosctrings qu'on utilise.
cf. https://google.github.io/styleguide/pyguide.html#383-functions-and-methods et https://docs.astral.sh/ruff/rules/blank-lines-between-header-and-content/

Si tu tentes de faire un `mkdocs serve` et de regarder la fonction, tu verras que ça rend pas très bien.

Reply:
La manière dont j'avais fixé était mauvaise, c'est fait :)